x-pack/plugins/security_solution/public/common/components/event_details/cti_details/helpers.test.tsx

```diff
@@ -356,3 +357,24 @@ describe('getEnrichmentFields', () => {
     });
   });
 });
+
+describe('getEnrichmentIdentifiers', () => {
+  it(`return feed name as feedName if it's present in enrichment`, () => {
+    expect(
+      getEnrichmentIdentifiers({
+        'matched.id': [1],
+        'matched.field': ['matched field'],
+        'matched.atomic': ['matched atomic'],
+        'matched.type': ['matched type'],
+        'feed.name': ['feed name'],
+        'indicator.provider': ['provider name'],
```

Comment:
do we use the `indicator.provider` enrichment identifier in Kibana? trying to understand if we need to leave it here considering it's a mapped field just like most others and it would work on the timeline anyway. if it's not used on the row renderer UI, there might not be an urgent reason to keep it here